doc/lsst.pex.config/inspecting-configs.rst

```diff
@@ -0,0 +1,29 @@
+.. py:currentmodule:: lsst.pex.config
+
+#########################
+Inspecting configurations
+#########################
+
+Field access
+------------
+
+Iterating through a `Config` instance yields the names of the `Field` attributes it contains.
+The `Config` class also supports many dictionary-like methods: `~Config.keys`, `~Config.items`, `~Config.iterkeys`, `~Config.iteritems`, and `~Config.itervalues`.
+
+History
+-------
+
+The `Config.history` attribute contains the history of all changes to the `Config` instance's fields.
+Each `Field` instance also has a history.
+The `Field.formatHistory` method displays the history of a given `Field` in a more readable format.
```

Comment:
The link to `Field.formatHistory` is broken in the html tarball you uploaded.

Also, if `Field.formatHistory` is the public entry point for all of the functionality in the `lsst.pex.config.history` module, perhaps that module should not be included in the public docs?